Octokit/Models/Request/SearchIssuesRequest.cs

```diff
@@ -337,4 +336,74 @@ public enum IssueTypeQualifier
         [Parameter(Value = "issue")]
         Issue
     }
+
+    public class RepositoryCollection : IEnumerable<string>
```

Comment:
The only reason it implements `IEnumerable<string>` is because it's needed for collection initializers to work. I had originally implemented `ICollection<string>`, but landed on `IEnumerable` in the original implementation because it didn't allow simple strings.